test/testdata/lsp/completion/snippet_before_variable.A.rbedited

```diff
@@ -0,0 +1,22 @@
+# typed: true
+
+class A
+  extend T::Sig
+  sig { params(x: Integer).void }
+  def self.example(x)
+    puts(x + 1)
+  end
+end
+
+begin
+  A.
+  # ^ completion: example, ...
+  # ^ apply-completion: [B] item: 0
+
+  x = nil # error: does not exist
+
+  A.example(${1:Integer})${0}
+  # ^ completion: example, ...
+  # ^ apply-completion: [A] item: 0
```

Comment:
This shows what the intended behavior is, and what it actually is when the completion happens at the end of a file or in a spot where the next token doesn't look like a method name.